vizro-core/tests/unit/vizro/models/_components/test_ag_grid.py

```diff
@@ -47,17 +49,25 @@ def test_mandatory_figure_missing(self):
         with pytest.raises(ValidationError, match="field required"):
             vm.AgGrid()
 
-    def test_wrong_captured_callable(self, standard_dash_table):
-        with pytest.raises(ValidationError, match="CapturedCallable mode mismatch"):
-            vm.AgGrid(figure=standard_dash_table)
-
-    def test_failed_ag_grid_with_no_captured_callable(self, standard_go_chart):
-        with pytest.raises(ValidationError, match="must provide a valid CapturedCallable object"):
+    def test_captured_callable_invalid(self, standard_go_chart):
```

Comment:
These tests for each model aren't strictly required now we have tests in `test_types.py` but I thought safest to leave them here and have somewhat duplicated tests, since these are better integration tests than the ones in `test_types`.

Reply:
At some point when we think about subclassing AgGrid, Table and Graph from Figure, we could probably kill all these duplicated tests and just test for these in Figure right?

Reply:
Probably yes, but I feel like there is also value in having good integration tests that run more "real" examples. Otherwise our tests are probably a bit too theoretical and will test the underlying mechanism of `Figure` but not whether it actually works in practice with `AgGrid` and there's a bit of a gap in testing coverage where things could conceivably go wrong and not be caught.

So if we remove them then we should make sure that there's good coverage of these models in our integration tests and/or QA tests.